src/views/EnrollmentCheckIn/components/CompletedCheckIn/index.jsx

```diff
@@ -1,13 +1,11 @@
 import { Button, Grid, Typography } from '@mui/material';
 import { useUser } from 'hooks';
 import { Link } from 'react-router-dom';
-import { gordonColors } from 'theme';
 // @TODO CSSMODULES - outside directory
```

Comment:
You should be able to remove this comment now, since we've added CSS modules to this